src/unix/process.c

```diff
@@ -453,13 +453,13 @@ int uv_get_children_pid(pid_t ppid, uint32_t** proc_list, int* proc_count) {
   if (fp == NULL)
     return 127;
 
-  while (getline(&line, &len, fp) >= 0) {
+  while (fscanf(fp, "%u", &match_pid) > 0) {
```

Comment:
I'll just cast from `int` to `uint32_t` below. Thought this would be more elegant. Thx again.